src/plugins/requestAuthPlugin.js

```diff
@@ -200,8 +205,8 @@ export const getAuthenticate = authorizeRequest$ => (request, reply) => {
 			request.log(['info', 'auth'], 'Request authenticated');
 		})
 		.subscribe(
-			({ state: { oauth_token } }) => {
-				const credentials = oauth_token;
+			({ state: { MEETUP_MEMBER, oauth_token } }) => {
```

Comment:
this could be refactored to read from MEETUP_MEMBER_DEV when using dev endpoints...